File: authenticate/src/main/java/com/suntelecoms/authenticate/util/Animate.java
package com.suntelecoms.authenticate.util;

import android.annotation.TargetApi;
import android.graphics.drawable.AnimatedVectorDrawable;
import android.os.Build;

import androidx.appcompat.widget.AppCompatImageView;

/**
 * @Author Moustapha S. Dieme ( Djvmil_ ) on 3/12/20.
 */
public class Animate {

    @TargetApi(Build.VERSION_CODES.M)
    public static void animate(AppCompatImageView view, AnimatedVectorDrawable scanFingerprint) {
        view.setImageDrawable(scanFingerprint);
        scanFingerprint.start();
    }
}

File: authenticate/src/main/java/com/suntelecoms/authenticate/util/Animate.kt
package com.suntelecoms.authenticate.util

import android.annotation.TargetApi
import android.graphics.drawable.AnimatedVectorDrawable
import android.os.Build
import android.widget.ImageView

/**
 * @Author Moustapha S. Dieme ( Djvmil_ ) on 3/12/20.
 */
object Animate {
    @JvmStatic
    @TargetApi(Build.VERSION_CODES.M)
    fun animate(view: ImageView, scanFingerprint: AnimatedVectorDrawable) {
        view.setImageDrawable(scanFingerprint)
        scanFingerprint.start()
    }
}